CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+there are certain types of pull requests (like changing `README.md`) that the continuous
+integration tests do not evaluate that a reviewer might merge regardless of continuous integration
+test status. In any case, generally most reviewers wait until everything passes before merging.
+
+If possible, please create a set of small self-contained pull requests rather than a single
+massive pull request. If a pull request consists of a sequence of well-defined steps, the best
+approach is to keep the *one step = one commit* approach.
+
+Avoid merge commits in pull requests - use *git rebase* to get rid of them. This helps us keep git
+tree clean and helps to make it easy to understand historical changes done in past commits.
+
+Contributing Content
+---
+
+If you are considering contributing content to the project, thank you for your willingness to do so.
+There are some guidelines and rules to consider when you are contributing content:
+
+1. There should only be 1 configuration change per XCCDF rule. This is to not only help us meet U.S. Government expectations, but it makes our content easier to use and tweak for the end users.
+1. Create variables when a configuration change can be multiple different values. For example if a rule says to "create a 16 character password", create a variable that by default is set to 16 and add other possible password lengths. This is not just to make the content more usable, but not everyone's computing environments require a 16 character password.
+1. It is always better to fix an issue in external upstream open source projects before contributing, fixing, or breaking content here. For example if the `gdm` rpm reports changed permissions when the `rpm_verify_permissions` rule runs, the fix needs to be provided in the `gdm` rpm not by changing or disabling the content. Another example of this is adding a configuration option like to `pam_faillock` and `pam_pwquality` to make it easy for centralized accounts to ignore local settings which may be governed by the centralized account management tool. Once the patches are merged in the upstream project, you should then open a new pull request in this project with the changes. Following this approach not only ensures that it is easier for others to securely configure their systems, but it makes other upstream projects better and easier to use.
+1. Ask before you disable or remove content because you feel that they are not security relevant. They might actaully be security relevant in ways that you might not realize. This way we all learn something.
+1. Products or content that have been previously retired should stay retired unless there is a heavy commitment to support that content.
+1. While this is an open source project, contributing content from projects that have been taken over by a hacker is a no-go. Not only can you or the project not guarantee that that project is truly cleaned up, but doing so diminishes the trust and provenance of this project.
```

Comment:
I feel that this paragraph is trying to say something, but I have no idea what. When one contributes rule descriptions, checks and remediations, those should be backed up by tests, and they get an independent review, so the source shouldn't matter, and it may not even be known.
What are other relevant scenarios here that this statement aims to address?